get_models_and_test.py

```diff
@@ -118,19 +134,13 @@ def mean_of_values_at_indices(arr, indices):
                             P_inf_counter = 0
                             cdf_counter = 0
                             
-                            
-                            multi_lstm_model = toy_models.get_fitted_model(data_input=data_config, 
-                                                                            model_name=model_name, 
-                                                                            epochs=epochs, 
-                                                                            force_retrain = force_retrain_models, 
-                                                                            lstm_units=lstm_size,
-                                                                            qth=0.5)
-
-
+                           
                             for _ in range(number_of_tests):
                                 X, y_label = training_generator.__getitem__(0)
-                                Y_pred = multi_lstm_model.predict(X)
-
+                                if isinstance(model, DQNLSTM):
+                                    Y_pred = model.model.predict(X)  # Use model.model for DQNLSTM instances
```

Comment:
Rather that doing this, can you create a method in your DQNLSTM class that looks something like this:
```
    def predict(self, *args, **kwargs):
        return model.predict(*args, **kwargs)
```

Reply:
This will make your class a bit like an [adaptor](https://en.wikipedia.org/wiki/Adapter_pattern)

Reply:
Done.